### PR TITLE
Add methods about the MTCS commands

### PR DESCRIPTION
**Pull request by fdelcampo**
Add methods:
- connect_to_mtcs_intance
- maintel_command
- maintel_docstrings
- unavailable_mtcs_client

Change URL for the docstrings:
- /aux/docstrings
- /main/docstrings

### CODE REVIEW

File: commander/tcs.py [hunk 1]
```diff
@@ -2,10 +2,14 @@
 provides the endpoints to request a heartbeat."""
 from aiohttp import web
 
-# from lsst.ts.observatory.control.auxtel import ATCS
-import lsst.ts.observatory.control.auxtel as auxtel
+from lsst.ts.observatory.control.auxtel import ATCS
+
+# import lsst.ts.observatory.control.auxtel as auxtel
```
Comment (sebastian-aranda) [inline commander/tcs.py:7]: This line should be removed

File: commander/tcs.py [hunk 2]
```diff
@@ -72,10 +76,68 @@ async def auxtel_docstrings(request):
...
+
+        methods = [
+            func
+            for func in dir(mtcs_client)
+            if callable(getattr(mtcs_client, func)) and not func.startswith("__")
+        ]
+        docstrings = {m: getattr(mtcs_client, m).__doc__ for m in methods}
+        return web.json_response((docstrings), status=200,)
+
+    def unavailable_mtcs_client():
```
Comment (sebastian-aranda) [inline commander/tcs.py:128]: I would move this method between `connect_to_mtcs_intance` and `maintel_command`, near line 88

**Review by sebastian-aranda — CHANGES_REQUESTED** [review]
It looks good, but I'm missing the tests. Once you have that ready I can approve the PR.

**Review by areyesd14 — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**